client/src/pages/Scores/Scores.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react'
+import { RouteComponentProps } from '@reach/router'
+import { Game } from '../../types/Game'
+import './Scores.scss'
+import Button from '../../components/Button'
+import { GameStatus } from '../../types/GameStatus'
+import { RoundResult } from '../../types/RoundResult'
+import { Player } from '../../types/Player'
+import ScoreBoard from '../../components/ScoreBoard'
+
+export interface ScoresProps extends RouteComponentProps {
+  game: Game
+  currentPlayerId: string
+  isGameHost: boolean
+  result?: RoundResult | undefined
+  startGame: () => void
+}
+
+const Scores: FC<ScoresProps> = (props: ScoresProps) => {
```

Comment:
Unecessary to annotate props type since it cannot have any other types than `ScoreProps` (because of `const Scores: FC<ScoresProps> =` )

```suggestion
const Scores: FC<ScoresProps> = props => {
```

Also, i am still on team deconstruct everything

```suggestion
const Scores: FC<ScoresProps> = ({ game, currentPlayerId, isGameHost, result, startGame }) => {
```

Reply:
✅